Tools/MiniBrowser/Configurations/Base.xcconfig

```diff
@@ -79,3 +79,6 @@ FRAMEWORK_SEARCH_PATHS[sdk=macos*] = $(inherited) $(WK_QUOTED_OVERRIDE_FRAMEWORK
 SDKROOT = macosx.internal;
 
 CODE_SIGN_IDENTITY = -;
+
+WK_PROCESSED_XCENT_FILE=$(TEMP_FILE_DIR)/$(FULL_PRODUCT_NAME).entitlements
+WK_USE_RESTRICTED_ENTITLEMENTS = $(USE_INTERNAL_SDK);
```

Comment:
Is this used anywhere? The script seems to use the presence of WebKitAdditions to decide whether to add additional entitlements. Which is fine, but just means we can probably omit this.

Reply:
I'll remove it.